core/server/master/src/main/java/alluxio/master/AlluxioSecondaryMaster.java

```diff
@@ -77,19 +82,30 @@
 
   @Override
   public void start() throws Exception {
-    mRegistry.start(false);
```

Comment:
we don't need to update the `mRegistry`?

Reply:
secondary_master never serves, so it doesn't need to start the masters. The *only* thing that secondary_master ever does is replay the journal, write checkpoints, and delete old journal files